index.html

```diff
@@ -185,5 +185,38 @@ <h1>Skills</h1>
                  </ul>
              </div>
           </section>
+          <footer class="footer-items">
+             <div class="feedback">
+              <p class="feedback-text">
+                  I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+              </p>                    
+              <form action="https://formspree.io/your-email" method="POST">
+                  <input class="form-input" type="text" id="fullname" name="fullname" placeholder="Full name" maxlength="30" required>
+                  <input class="form-input" type="email" id="email" name="email" placeholder="Email address" required>
+                  <textarea id="message" name="message" rows="5" placeholder="Write me something..." maxlength="500" required></textarea>                        
+                  <input class="submit-btn" type="submit" value="Get in touch">
+              </form>
```

Comment:
- [ ] Excellent work on using flexbox to layout the form elements! However, there is an adjustment needed to match the design guidelines. Instead of using the `margin` property to create gaps between the elements, I recommend using the `gap` property to achieve a more consistent and visually pleasing layout. Please update your code accordingly to ensure the gaps between the form elements match the design. This will enhance the overall aesthetics of the form. Keep up the great work! 👍🌟

![image](https://github.com/Obimbo07/Portfolio-2/assets/55840999/6ce105cf-06d2-43a1-8c1c-d8b778663bda)